### PR TITLE
frontend in Go

### PR DESCRIPTION
**Pull request by neelance**
Hello Andrew,

you said at dotGo that the JavaScript part is kind of diminishing the fun of working on sigourney and that GopherJS might be a solution to that. I took you by your word and wrote a Go version of the frontend. I applied some optimisations and the protocol is now shared between frontend and backend. There is still a lot of room for improvements, e.g. some libraries by the GopherJS community may be used to wrap the JavaScript calls in a type-safe fashion. I hope you like it!

Cheers,
Richard

### CODE REVIEW

**Comment by thomasf** [comment]
I agree with moving away from the browser, we need a euro rack module programmed in Go :)

**Comment by shurcooL** [comment]
Whoa! This is awesome.

File: README.md [hunk 1]
```diff
@@ -23,6 +23,13 @@ Third, download (or update) and build Sigourney and its dependencies:
 
 	$ go get -u github.com/nf/sigourney
 
+Fourth, install [GopherJS](https://github.com/gopherjs/gopherjs):
+
+	$ go get -u github.com/gopherjs/gopherjs
+
+Fifth, build the frontend:
+
+	$ gopherjs build $GOPATH/src/github.com/nf/sigourney/static/script.go -o $GOPATH/src/github.com/nf/sigourney/static/script.js
```
Comment (shurcooL) [inline README.md]: You could make this step easier by turning it into a `go:generate` directive, then you just need to run a standard `go generate ./...` command to update the package.

**Comment by nf** [comment]
I just committed 3c2d0352804d22c3d1f9d8e90766f0b129dc1f74 announcing (very belatedly) that Sigourney is unmaintained, so I should probably close this PR now. Sorry for being so slack about this, and thanks for your interest in the project. <3

**Pull request closed without merging**